doc/man5/config.pod

```diff
@@ -438,6 +438,16 @@ generators will use.  Other random bit generators ignore this name.
 This sets the property query used when fetching the random bit generator and
 any underlying algorithms.
 
+=item B<seed>
+
+This sets the randomness source that should be used.  By default B<SEED-SRC>
+will be used outside of the FIPS provide.  The FIPS provider, instead, uses the
+L<ossl_rand_get_entropy(3)> call backs.
```

Comment:
We shouldn't reference internal API functions from the public docs.

Reply:
Hmmm, not sure what to reference then.  I've changed it to _The FIPS provider uses call backs
to access the same randomness sources from outside the validated boundary_.